app/models/ncr/work_order.rb

```diff
@@ -29,6 +29,10 @@ class WorkOrder < ActiveRecord::Base
     validates :expense_type, inclusion: {in: EXPENSE_TYPES}, presence: true
     validates :vendor, presence: true
     validates :building_number, presence: true
+    validates :rwa_number, format: {
+      with: /[a-zA-Z][0-9]{7}/,
```

Comment:
@phirefly Can we look at a list of RWAs, or ask someone to double-check that this format is correct?